source/_integrations/tedee.markdown

```diff
@@ -25,6 +25,7 @@ This integration interacts with your [Tedee](https://tedee.com) locks by communi
 
 - You will need the bridge to add your locks using this integration.
 - You need to have the local API enabled.
+- Bridge firmware needs to be at least version `2.2.18086` for push updates to work without errors
```

Comment:
```suggestion
- The bridge firmware needs to be at least version `2.2.18086` for push updates to work without errors.
```
Tiny tweak. Rephrased this to be a full sentence to align the style with the other list items.